test/Microsoft.AspNetCore.JsonPatch.Test/project.json

```diff
@@ -16,30 +17,15 @@
           "version": "1.0.0-*",
           "type": "platform"
         },
-        "moq.netcore": "4.4.0-beta8",
-        "System.Diagnostics.Process": "4.1.0-*",
         "System.Diagnostics.TraceSource": "4.0.0-*"
       },
-      "imports": [
-        "dnxcore50",
-        "portable-net451+win8"
-      ]
+      "imports": "dotnet5.4"
```

Comment:
Yeah, the castle core package they are referencing is busted. But it gets us on RTM packages and off the custom builds we have so I guess it's worth the effort for now.